app/Http/Controllers/MeetupController.php

```diff
@@ -71,6 +73,6 @@ public function sendEmailOnMeetupCreation(Meetup $meetup)
         Mail::send('emails.meetup-created', compact('user', 'meetupDetails'), function ($message) use ($user, $recipients) {
```

Comment:
As is, I think it's okay because the `$recipients` variable holds the emails of all the intended recipients of a meetup notification email: the admins and the user who created the meetup. The user's email is just appended to the end of the array after the admins' emails.

However, I didn't know of the `cc()` method at the time. I'll separate the addressees into primary (the user => to) and secondary (admins => cc).

But that's not the cause of the error. That's because the ADMIN_EMAILS variable (which was sent earlier today on Slack) is not set on Heroku for the PR build, but it's set for the `staging` build.

Do I have access to the PR build settings as well?